utils/utils.js

```diff
@@ -3,7 +3,7 @@ const CharModel = require('../models/Character');
 const UserModel = require('../models/User');
 const EventModel = require('../models/Event');
 const GuildModel = require('../models/Guild');
-const config = require('../handlers/config.js');
+const Config = require('../handlers/config.js');
```

Comment:
don't do this require .... `Config` is coming from the global.Config (from config.json in index.js)